bottlerocket/agents/src/sonobuoy.rs

```diff
@@ -57,6 +69,8 @@ pub async fn run_sonobuoy(
     // TODO - log something or check what happened?
     ensure!(status.success(), error::SonobuoyRunSnafu);
 
+    results.other_info = Some("Checking status".to_string());
+    info_client.send_test_update(results).await?;
```

Comment:
Same as previous comment. Do we want to bring down the house of cards if the Kubernetes API is unreachable for a millisecond?